peering_concourse.tf

```diff
@@ -1,13 +1,21 @@
+resource "aws_vpc_peering_connection" "peering" {
+  count       = local.roles[0] == "master" ? 1 : 0
+  peer_vpc_id = data.terraform_remote_state.aws_concourse.outputs.route_tables[0].vpc_id
```

Comment:
This looks a little odd. Does aws-concourse not output its VPC info? If not, maybe it should (as should all other VPCs in due course to support this peering)? I wouldn't want this pattern to be copied in future and result in other VPCs outputting route tables in order to expose their VPC IDs.

Reply:
Resolved